### DISCUSSION TITLE
2005/08/13/%E5%8F%B0%E9%A3%8E%E4%B8%AD%E7%9A%84%E4%B8%8A%E6%B5%B7

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#46] [Announcements]
# 2005/08/13/%E5%8F%B0%E9%A3%8E%E4%B8%AD%E7%9A%84%E4%B8%8A%E6%B5%B7

Quote 台风中的上海

https://blog.qizhen.xyz/2005/08/13/%E5%8F%B0%E9%A3%8E%E4%B8%AD%E7%9A%84%E4%B8%8A%E6%B5%B7

**Comment by ruanqizhen** [comment]
Tian
2005年08月18日 20:31 编辑
hoho, Matrix reloaded~~~

回复
米拉
2005年08月15日 23:11 编辑
我觉得象天煞

回复
Qian
2005年08月13日 20:35 编辑
更象台风前的上海

回复
JING
2005年08月13日 18:53 编辑
感觉很象the end of the world~~
